src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/clients/CamundaServiceApi.java

```diff
@@ -90,4 +90,12 @@ void assigneeTask(@PathVariable("task-id") String id,
     @ResponseBody
     Map<String, CamundaVariable> getVariables(@PathVariable("task-id") String id);
 
+    @GetMapping(
+        value = "/task/{task-id}/localVariables",
+        produces = APPLICATION_JSON_VALUE
+    )
+    @ResponseBody
+    Map<String, CamundaVariable> getLocalVariables(@PathVariable("task-id") String id);
```

Comment:
I don't think we need this endpoint for the task mapper instead we could use:
`/task/{task-id}/variables`
which should return all variables, the reason for this is performance, when we map a task we need everything in camunda to map it to our Task object, in this case: we could have some variables that could only be retrieved through the /variables endpoint so we will have to query both endpoints `/localVariables` and `/variables`
so might be better to use `/variables` as it should contain all variables, not just the local ones.